octoprint_telegram/telegramCommands.py

```diff
@@ -121,15 +124,15 @@ def cmdSuperGif(self,chat_id,from_id,cmd,parameter): #GWE 05/05/2019 add command
 				try:
 					sendOneInLoop = False
 					self._logger.info("Will try to create a super gif")
-					if self.main._plugin_manager.get_plugin("multicam") and self.main._settings.get(["multicam"]):
+					if self.main._plugin_manager.get_plugin("multicam",True) and self.main._settings.get(["multicam"]):
 						try:
 							curr = self.main._settings.global_get(["plugins","multicam","multicam_profiles"])
-							self._logger.error("MUUUUUUUUUUUUUUUUUULTICAM  "+ str(curr))
+							self._logger.error("multicam_profiles : "+ str(curr))
```

Comment:
Are you sure you want these to be error level logs? I suggest to change those to debug

Reply:
yep you are right I change that directly